tests/testthat/test-base-plot.R

```diff
@@ -308,6 +308,31 @@ describe("Base plots", {
     vdiffr::expect_doppelganger("Character interpretation of numeric group_level", plt2)
 
   })
+
+  it("Change CI interval format [PMF-PLOT-025]", {
```

Comment:
I think you're holding off on reqs because things are up in the air, but is it worth adding the test ID if that's the case?